mathics/builtin/system.py

```diff
@@ -871,3 +871,30 @@ def eval_with_symbol(self, symbol, evaluation: Evaluation) -> Integer:
         else:
             gc.collect()
             return Integer0
+
+
+class Breakpoint(Builtin):
+    """
+    ## <url>:trace native symbol:</url>
+
+    <dl>
+      <dt>'Breakpoint[]'
+      <dd> Invoke a Python breakpoint. By default, the Python debugger \
+           (pdb) is loaded. For loading other debuggers, the Python environment \
+           variable `PYTHONBREAKPOINT` can be utilized.
+    </dl>
+
+    >> (* Test with a disabled breakpoint. *)
+    >> In[1]:= SetEnvironment["PYTHONBREAKPOINT" -> "mathics.disabled_breakpoint"]
```

Comment:
```
    >> SetEnvironment["PYTHONBREAKPOINT" -> "mathics.disabled_breakpoint"];
    >> Breakpoint[]
       = ...

```

Reply:
@mmatera as I suggested to rocky, if you don't mind, you could use the +/- option in the review to suggest the changes, which I can apply directly. That way, your contribution is also acknowledged, and it's easier for me as well...

Reply:
Regarding the URL, I will leave it to you both to decide the best course and advise me on the change.

Reply:
> @mmatera as I suggested to rocky, if you don't mind, could you please use the +/- option in the review to suggest the changes, which I can apply directly. That way, your contribution is also acknowledged, and it's easier for me as well...
> 

@aravindh-krishnamoorthy, actually I did it in that way: I added the comment just in the line where your test is localized.

> Please see step 8 (as of today) here for details: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request